### DISCUSSION TITLE
on save keep the file open

### DISCUSSION THREAD

**Discussion by onweru** [sveltia/sveltia-cms#176] [Q&A] [answered]
When I make edits on a document and save, I'm redirected to the collection page. Is it possible to stay on the same resource I'm editing?

**Comment by kyoshino** [comment]
It’s intentional, because I thought that 99% of the time people would just want to close the editor after saving the entry. But I could add a setting to change the default behaviour.

**Reply by onweru** [reply]
Please add a setting for that 🙏🏼 .

**Comment by kyoshino** [comment, marked as answer]
An option added in [v0.36.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.36.0) ✅
